projects/smoldot-browser-demo/tsconfig.json

```diff
@@ -1,18 +1,18 @@
 {
```

Comment:
Im not sure which change does the following, but when building smoldot-browser-demo and multiple-network-demo, src directory is included in the dist directory which did not use to do before

Reply:
Fixed